lib/modules/hosts/pixiv.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import { Host } from '../../core/host';
+
+export default new Host('pixiv', {
+	name: 'pixiv',
+	domains: ['pixiv.net'],
+	logo: '//www.pixiv.net/favicon.ico',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?pixiv\.net\/member_illust\.php.*[?&]illust_id=(\d+)/i).exec(href),
```

Comment:
This can be simplified:

```js
detect: ({ pathname, search }) => pathname === '/member_illust.php' && getUrlParams(search).illust_id,
```

Then add `import { getUrlParams } from '../../utils';` at the top and change the below to `handleLink(href, id) {`